### PR TITLE
Kill Streaks

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Added kill streaks, voiced by Cheridan.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Yes finally

**Comment by GunHog** [comment]
Kor is meme-ing again, someone reboot him, for he has crashed!

**Comment by Grazyn** [comment]
can we bring back public k/d stats too?

**Comment by evsey9** [comment]
yes

**Comment by Cyberboss** [comment]
#S P E E D M E R G E

**Comment by Jazaen** [comment]
Hm, has Cheridan also done Reaper voicelines?

**Comment by Niknakflak** [comment]
Who made this voice finals jesus christ ew

**Comment by calzilla1** [comment]
No killamanjaro
Apply your self

**Comment by MMMiracles** [comment]
I'm glad were finally taking the game in the appropriate direction.

**Comment by bgobandit** [comment]
post audio pls

**Comment by optimumtact** [comment]
This isn't funny kor

**Comment by KorPhaeron** [comment]
@optimumtact 

April fools never is

File: code/datums/mind.dm [hunk 1]
```diff
@@ -63,6 +63,34 @@
 	var/isholy = FALSE //is this person a chaplain or admin role allowed to use bibles
 
 	var/mob/living/enslaved_to //If this mind's master is another mob (i.e. adamantine golems)
+	var/kill_streak = 0 //How many kills we have
+
+/datum/mind/proc/kill_streak_act()
+	var/sound
+	switch(kill_streak)
+		if(3)
+			world << "<B>[name] is on a killing spree!</B>"
```
Comment (ChangelingRain) [inline code/datums/mind.dm]: use send_to_playing_players() instead of world <<

File: code/datums/mind.dm [hunk 2]
```diff
@@ -63,6 +63,34 @@
...
+			sound = 'sound/killstreaks/Unstoppable.ogg'
+		if(11)
+			world << "<B>[name] is WICKED SICK!!</B>"
+			sound = 'sound/killstreaks/HOLYSHIT.ogg'
+		if(13)
+			world << "<B>[name] is GODLIKE!</B>"
+			sound = 'sound/killstreaks/GODLIKE.ogg'
+	if(sound)
+		for(var/mob/M in player_list)
+			M << sound
```
Comment (ChangelingRain) [inline code/datums/mind.dm]: same for this

**Comment by BeeSting12** [comment]
Time to rack up a new high score!

File: code/datums/mind.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/datums/mind.dm]: This will probably be useful for me to know later but I probably won't update this PR since it is never going to be merged

**Comment by Doppeltongue** [comment]
You forgot the "SHUT DOWN" part. How could you, Kor?

**Comment by Repukan** [comment]
>DotA

More like nota

**Comment by Niknakflak** [comment]
I made a preview of some alternative sounds we could use.  If people like it, I can make the rest of the sounds.

https://file.house/q5Hz.ogg

**Comment by Ausops** [comment]
it is time

**Comment by ShizCalev** [comment]
Is there a way to make this not global and only play for people within view of whomever triggered it? It seems really meta for me to know that Bob Johnson's killed 9 people while I'm off in a secluded tunnel in maint with no radio.

**Pull request closed without merging**